packages/ui/pages/logout.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect } from "react";
```

Comment:
I don't think we need a logout page. There is already a logout button. So this file can be deleted. :)

Reply:
Yup, you are right. Deleted the file and passed the logout handler from MainNav and MainNavMobile components just as the way closeDrawer was passed.